code/game/objects/items/oddities.dm

```diff
@@ -266,3 +266,16 @@
 		STAT_ROB = 6,
 		STAT_VIG = 6,
 	)
+
+/obj/item/weapon/oddity/rare/golden_cup
```

Comment:
Maybe have this not be-able to get annomlies, so it cant be like `Oh wow their was fungus in the cup...`
just a suggestion

Reply:
Gold cup lottery.